### PR TITLE
Quest3 streamer + recreate P1 demo with Quest3

### PR DESCRIPTION
**Pull request by KavitShah1998**

### CODE REVIEW

File: perception_and_utils_root/perception_and_utils/perception/detector_wrappers/object_detector.py [hunk 1]
```diff
@@ -223,7 +226,9 @@ def process_frame(self, img_frame: np.ndarray) -> Tuple[np.ndarray, Dict[str, An
         )
         return updated_img_frame, object_scores
 
-    def process_frame_offline(self, img_frame: np.ndarray):
+    def process_frame_offline(
+        self, frame: DataFrame
+    ) -> Tuple[np.ndarray, Dict[str, Any]]:
```
Comment (zephirefaith) [inline perception_and_utils_root/perception_and_utils/perception/detector_wrappers/object_detector.py:231]: What was the difference b/w `process_frame` and `process_frame_offline`?

File: perception_and_utils_root/perception_and_utils/perception/detector_wrappers/object_detector.py
Comment (zephirefaith) [inline perception_and_utils_root/perception_and_utils/perception/detector_wrappers/object_detector.py:1]: I think this is a very specific kind of object-detector, right? Maybe rename this file to reflect the actual usage? This is kind of a heuristic/bbox-IOU-based-hand-object-detector? Maybe `hand_object_iou_detector.py`

**Review by zephirefaith — APPROVED** [review]
Few notes. Given this works I am happy to merge this. If the tests are not broken, please make sure they pass.

Some of our interfaces are maybe more complex than necessary, let's address that as a BE later.

File: perception_and_utils_root/perception_and_utils/perception/detector_wrappers/object_detector.py (hunk 1 above)
Comment (KavitShah1998) [reply perception_and_utils_root/perception_and_utils/perception/detector_wrappers/object_detector.py:231]:
```
            # object-detection + checking for intersection b/w all objects and hand
            # is a computationally expensive operation (happens inside
            # _get_scored_object_detections()).
            # For efficiency, in this code, we turn off object detection for the objects
            # that have already been detected and their place location has been recorded
            # This is done by removing the object_name from the labels and regenerating
            # the prompts for the object detector.
```

In the offline version, we disable the `_get_scored_object_detections` which is computationally expensive for detected and processed objects

File: perception_and_utils_root/perception_and_utils/perception/detector_wrappers/object_detector.py
Comment (KavitShah1998) [reply perception_and_utils_root/perception_and_utils/perception/detector_wrappers/object_detector.py:1]: It is not exactly / explicitly IOU as Tushar and Vyshnav have an explicit IOU based method
```
Simple heuristic where we detect only the objects of interest and score current
frame depending on the % of pixels occupied by the object in the image frame and
proximity of bbox to image center. This is then passed to the EMRoF algorithm.
```
Any other naming suggestion that you think fits the tag well?

**Comment by KavitShah1998** [comment]
The PR has bad code structure. Due to limited bandwidth, we plan to fixing the code structure as BE in December (can be tracked here - https://github.com/facebookresearch/spot-sim2real/issues/246)

**Pull request merged**